spec/acceptance/api/cms_file_spec.rb

```diff
@@ -21,7 +21,7 @@
       parameter :attachment, 'The Attachment'
 
       let(:section_id) { resource.section_id }
-      let(:attachment) { fixture_file_upload('/wide.jpg',' image/jpeg') }
+      let(:attachment) { fixture_file_upload('wide.jpg',' image/jpeg') }
```

Comment:
This changed from `File.join(self.class.fixture_path, path)` to `Pathname.new(self.class.fixture_path).join(path)` and the latter treats `/wide.jpg` as an absolute path (rightfully, IMO).

![image](https://github.com/3scale/porta/assets/1270328/a06c98d5-c59c-41f9-a278-6ee0379f0ab6)